lib/Models/addUserFiles.js

```diff
@@ -1,5 +1,8 @@
+'use strict';
+
 import addUserCatalogMember from './../Models/addUserCatalogMember';
 import createCatalogItemFromFileOrUrl from './../Models/createCatalogItemFromFileOrUrl';
+import AsyncFunctionResultCatalogItem from './../Models/AsyncFunctionResultCatalogItem';
```

Comment:
Clever use of `AsyncFunctionResultCatalogItem` here, but maybe we should rename it, since this has nothing to do with async functions.